guides/v2.1/cloud/cdn/fastly-vcl-wordpress.md

```diff
@@ -11,62 +11,127 @@ functional_areas:
+
+
+#### To add the custom VCL snippet from the Admin UI
+
+1.  From the Magento Admin UI, navigate to **Stores** > **Configuration** > **Advanced** > **System** > **Fastly Configuration**.
+
+1.  Expand the **Custom VCL snippet** section.
+
+1.  Click **Create Custom Snippet**.
+
+1.  Complete the snippet form as shown in the following figure:
+
+    ![Create VCL Snippet]
+
+1.  After the page reloads, click **Upload VCL to Fastly** in the *Fastly Configuration* section.
+
+1.  After the upload completes, refresh the cache according to the notification at the top of the page.
+
+Fastly validates the updated version of the VCL code during the upload process. If the validation fails, edit your custom VCL snippet to fix the issue. Then,  upload the VCL again.
 
-Review the following values for the code to determine if you need to make changes:
 
-* `name`: Name for the VCL snippet. For this example, we used the name `wordpress`.
-* `priority`: Determines the order VCL snippets call. You want to set the priority to 5 to immediately run and check for URLs that should be redirected. This priority runs the snippet immediately and before any of the uploaded and default Magento VCL snippets (magentomodule) that have a priority of 50.
-* `type`: For this VCL, we use `recv`, which places it in the vcl_recv subroutine by below the boilerplate VCL and above any objects.
-* `content`: The code that runs. The code extracts the first part `mypath` of the path `/mypath/someotherpath`.  It then compares that path against the Edge Dictionary `wordpress_urls`. If a match is found, the visitor is redirected to the Wordpress backend.
+{: .bs-callout .bs-callout-info}
+Instead of manually uploading custom VCL snippets, you can add snippets to the `$MAGENTO_HOME/var/vcl_snippets_custom` directory in your environment. Snippets in this directory upload automatically when you click *upload VCL to Fastly* in the Admin UI. See [Automated custom VCL snippets deployment](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/CUSTOM-VCL-SNIPPETS.html#automated-custom-vcl-snippets-deployment) in the Fastly-Magento module documentation. 
 
-{:.bs-callout .bs-callout-info}
-The default VCL snippets you uploaded included a prepended name of `magentomodule_` with a priority of 50. For your custom VCL snippets, **do not use the `magentomodule_` name**. Also consider the priority of your custom snippets if they should override the default snippets.
 
-## Configure Wordpress {#wordpress}
+[Add Edge Dictionary Container]: {{site.baseurl}}/common/images/cloud/cloud-fastly-edge-dictionary-example.png
```

Comment:
I don't see this line and below on the preview here: https://devdocs.magedevteam.com/mae-MAGECLOUD-2758-fix-fastly-wordpress-redirect-example/guides/v2.3/cloud/cdn/fastly-vcl-wordpress.html

Is the preview out of date?

Reply:
Those are the link definitions for image references in the document -- for example, the image reference ![Add Edge Dictionary Container]  -- displays the file {{site.baseurl}}/common/images/cloud/cloud-fastly-edge-dictionary-example.png.

I'll add a code comment <!--Link definitions--> to clarify.

Reply:
Oh! Thanks for explaining that. I didn't know Markdown had reference-style links. No comment should be necessary since it's a core Markdown feature.